bin/convert-argv.js

```diff
@@ -94,14 +94,19 @@ module.exports = function(optimist, argv, convertOptions) {
 		configFileLoaded = true;
 	}
 
-	if(typeof options === "function") {
+	var isES6DefaultExportedFunc = (
+		typeof options === "object" && options !== null && typeof options["default"] === "function"
```

Comment:
`options["default"]` should be rewritten as `options.default`, it's just cleaner that way